praw/models/reddit/subreddit.py

```diff
@@ -1517,6 +1524,96 @@ def update_invite(self, redditor, permissions=None):
         self.subreddit._reddit.post(url, data=data)
 
 
+class Modmail(object):
+    """Provides modmail functions for a subreddit."""
+    def __call__(self,  # pylint: disable=invalid-name,redefined-builtin
+                 id=None, url=None):
+        """Get an individual conversation."""
+        return ModmailConversation(self.subreddit._reddit, id=id, url=url)
+
+    def __init__(self, subreddit):
+        self.subreddit = subreddit
+
+
+class ModmailConversation(RedditBase):
+    """A class for modmail conversations."""
+    STR_FIELD = 'id'
+
+    @classmethod
+    def parse(cls, data, reddit):
+        if data['conversation']:
```

Comment:
Also, should this condition `if data['conversation']` apply to the entire method? It appears that there are dependent attributes outside of it.

If it should apply to the entire method consider:

    if not conversation:
        return None

Although in what cases is conversation not set?